src/fides/api/ops/service/privacy_request/request_runner_service.py

```diff
@@ -416,41 +418,21 @@ async def run_privacy_request(
             _log_exception(exc, CONFIG.dev_mode)
             return
 
-        # Send erasure requests via email to third parties where applicable
-        if policy.get_rules_for_action(
-            action_type=ActionType.erasure
-        ) and can_run_checkpoint(
-            request_checkpoint=CurrentStep.erasure_email_post_send,
```

Comment:
Removing synchronous erasure emails, they will be batched just like consent emails.